mava/utils/environments/debugging_utils.py

```diff
@@ -25,6 +25,7 @@
     SwitchGameWrapper,
     TwoStepWrapper,
```

Comment:
Since this is not directly related to this PR, I suggest we do it in a different PR to avoid maybe breaking something that actually has nothing to do with this PR.

Reply:
Fair enough. I will create an issue for it.